internal/appsec/waf/waf.go

```diff
@@ -44,6 +44,14 @@ import (
 // Version wrapper type of the WAF version.
 type Version C.ddwaf_version
 
+// AtomicDuration can be used to perform atomic duration sums thanks to AtomicDuration.Add.
+type AtomicDuration uint64
+
+// Add atomically sums the current duration value with `ns` to create a new duration value.
+func (d *AtomicDuration) Add(ns uint64) {
+	atomic.AddUint64((*uint64)(d), ns)
+}
+
```

Comment:
Sorry for this untold coding style but we usually regroup types and their methods, and you inserted that code in the  middle of `Version` ^^ Please move it 🙏